backend/src/main/java/com/handong/rebon/review/application/ReviewService.java

```diff
@@ -55,6 +58,22 @@ public Long create(ReviewCreateRequestDto reviewCreateRequestDto) {
         return savedReview.getId();
     }
 
+    @Transactional
+    public void delete(ReviewDeleteRequestDto reviewDeleteRequestDto) {
+        Long reviewId = reviewDeleteRequestDto.getReviewId();
+        Long memberId = reviewDeleteRequestDto.getMemberId();
+
+        Review review = reviewRepository.findById(reviewId).orElseThrow(ReviewNotFoundException::new);
+        Member member = memberService.findById(memberId);
+
+        if (review.canDelete(member)) {
+            review.delete();
+            return;
+        }
```

Comment:
review에게 `delete()`만 호출하고 검증 작업은 delete 내부에서 처리해도 될 것 같아요

```suggestion
        review.delete(member);
```

```java
   public void delete(Member member) {
        if (this.member.isSame(member) || member.isAdmin()) {
            deleteContent();
        }
    }
```